### PR TITLE
m17n.1.0 - via opam-publish

### PR DESCRIPTION
**Pull request by whitequark**

### CODE REVIEW

File: packages/m17n/m17n.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "ocamlbuild"
+  "-classic-display"
+  "-use-ocamlfind"
+  "src_test/test_m17n.byte"
+  "--"
+]
+depends: [
+  "ocamlfind" {build}
+  "sedlex"
+  "uutf" {>= "0.9.4"}
```
Comment (dbuenzli) [inline packages/m17n/m17n.1.0/opam]: Could you turn this into an equality please. Next version of `uutf` will support `Uchar`.

File: packages/m17n/m17n.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "uunf"
```
Comment (dbuenzli) [inline packages/m17n/m17n.1.0/opam]: Here also could you contrain to the latest version.

File: packages/m17n/m17n.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "uucp"
```
Comment (dbuenzli) [inline packages/m17n/m17n.1.0/opam]: Again.

Comment (whitequark) [inline packages/m17n/m17n.1.0/opam]: Are these all breaking changes?

Comment (dbuenzli) [inline packages/m17n/m17n.1.0/opam]: Yes.

**Comment by whitequark** [comment]
@dbuenzli Constrained according to semver.

**Comment by yallop** [comment]
This isn't installable because there isn't yet a version of sedlex that supports OCaml 4.03.0.

**Comment by whitequark** [comment]
@yallop `new installable packages (1): m17n.1.0` ?

**Comment by yallop** [comment]
@whitequark: yes, I don't think the installability check fully take version constraints into account.  @AltGr might have more to say about that.

@alainfrisch, are you planning to make a 4.03-compatible release of sedlex?

**Comment by alainfrisch** [comment]
> @alainfrisch, are you planning to make a 4.03-compatible release of sedlex?

#6851

**Comment by yallop** [comment]
I've restarted the [4.03 job](https://travis-ci.org/ocaml/opam-repository/jobs/139734474).

**Comment by yallop** [comment]
The build succeeded with OCaml 4.03.0 and sedlex 1.99.3.

**Pull request merged**